src/Mjolnir.ts

```diff
@@ -153,7 +153,7 @@ export class Mjolnir {
             await logMessage(LogLevel.DEBUG, "Mjolnir@startup", "Loading protected rooms...");
             await this.resyncJoinedRooms(false);
             try {
-                const data = await this.client.getAccountData(PROTECTED_ROOMS_EVENT_TYPE);
+                const data: Object|null = await this.client.getAccountData(PROTECTED_ROOMS_EVENT_TYPE);
```

Comment:
this feels a bit overly explicit - the return type should be clear enough?

Reply:
Not in my tests.